manifests/init.pp

```diff
@@ -46,13 +39,13 @@
       }
 
       $pam_module  = $::architecture ? {
-        i386   => '/lib/security/pam_duo.so',
-        i686   => '/lib/security/pam_duo.so',
-        x86_64 => '/lib64/security/pam_duo.so'
+        'i386'   => '/lib/security/pam_duo.so',
+        'i686'   => '/lib/security/pam_duo.so',
+        'x86_64' => '/lib64/security/pam_duo.so'
       }
```

Comment:
linting thought: should there be a default fail in here for unknown/unhandled architectures?

Reply:
There are at least two most possible architectures that might be supported - arm and arm64. Neither are [officially supported](https://github.com/duosecurity/duo_unix/issues/84) yet. I could either add those two archs, or set `$pam_module` to `undef`, or have this fail loudly. I'm not inclined to undef, because that would need more checks elsewhere. So I'm choosing to have this fail loudly.

Reply:
Added a default in f69ffb3ad68984c68618e07d37a4af0ef253dd65.